azavea/cac-tripplanner
# Add accessibility flag

## Overview

Add `accessible` flag to destinations and events, to note if they are ADA accessible or not.


### Demo

Editing an event in the admin panel:
![accessible](https://user-images.githubusercontent.com/960264/33494374-0f9e3ecc-d691-11e7-88d7-5fc345993cfb.png)


## Testing Instructions

 * Run migrations
 * Should have new `accessible` field as a checkbox in the admin site for destinations and events
 * Flag should be returned in serialized destinations [here](http://localhost:8024/api/destinations/search?text=)

Closes #918.
